man/systemd.unit.xml

```diff
@@ -1226,6 +1226,22 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>ConditionEnvironment=</varname></term>
+
+          <listitem><para><varname>ConditionEnvironment=</varname> may be used to check whether a specific
+          environment variable is set (or if prefixed with the exclamation mark — unset) in the service
+          manager's environment block. The argument must either be a single word, or an assignment (i.e. two
+          words, separated by <literal>=</literal>). In the former case the environment is searched as left
+          hand side of an environment variable assignment. In the latter case, the exact assignment is looked
+          for with right and left hand side matching. Note that the environment block of the service manager
+          itself is checked, i.e. not any variables defined with <varname>Environment=</varname> or
```

Comment:
Maybe: "The argument may be a single word, to check if the variable with this name is defined in the environment block, or an assignment (<literal>name=value</literal>), to check if the variable with this exact value is defined."